### ISSUE TITLE
package not completed when installing via npm

### ISSUE THREAD

**Issue by ngaffa** [freepik-company/flaticon-uicons#3]
When installing flat icon via npm with this command 
````
npm i @flaticon/flaticon-uicons
````
The folder nodes_modules/@flaticon/flaticon-uicons doesn't contains these three packages: 
/css 
/svg 
/webfont 

it just contains this files : 
changelog.md
license.js
LICENSE.txt
package.json
README.md
svgo.config.js

**Comment by ngaffa** [comment]
i think , you shouldn't add src folder to .npmignore

**Comment by pascuflow** [comment]
Same

**Comment by ngaffa** [comment]
> Same, what other icon library did you guys switch to?

I stay with flaticon but I used with CSS links and import directly from theire endpoint :
@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap");
@import "customize-bootstrap.scss";
@import "../node_modules/bootstrap/scss/bootstrap";
@import url("https://cdn-uicons.flaticon.com/uicons-solid-rounded/css/uicons-solid-rounded.css");
@import url("https://cdn-uicons.flaticon.com/uicons-bold-straight/css/uicons-bold-straight.css");
@import url("https://cdn-uicons.flaticon.com/uicons-regular-rounded/css/uicons-regular-rounded.css");
@import url("https://cdn-uicons.flaticon.com/uicons-solid-straight/css/uicons-solid-straight.css");
@import url("https://cdn-uicons.flaticon.com/uicons-bold-rounded/css/uicons-bold-rounded.css");
@import url("https://cdn-uicons.flaticon.com/uicons-regular-rounded/css/uicons-regular-rounded.css");
@import url("https://cdn-uicons.flaticon.com/uicons-regular-straight/css/uicons-regular-straight.css");
@import url("https://cdn-uicons.flaticon.com/uicons-brands/css/uicons-brands.css");

**Comment by ngaffa** [comment]
i wish they can fixe the issue

**Comment by carloschneider** [comment]
A simple workaround while they don't fix it:

```sh
yarn add git+https://github.com/flaticon/flaticon-uicons.git
```

or

```sh
npm i git+https://github.com/flaticon/flaticon-uicons.git
```

**Comment by ngaffa** [comment]
> A simple workaround while they don't fix it:
> 
> ```shell
> yarn add git+https://github.com/flaticon/flaticon-uicons.git
> ```
> 
> or
> 
> ```shell
> npm i git+https://github.com/flaticon/flaticon-uicons.git
> ```

NICE IDEA :)

**Comment by jrmacia-freepik** [comment]
This problem has been fixed as of version 1.7.1

Thanks @carloschneider  for the temporary patch 😉
